packages/next-codemod/transforms/__testfixtures__/new-link/custom-component-child.output.js

```diff
@@ -6,8 +6,8 @@ function Comp({children}) {
 
 export default function Page() {
     return (
-        (<Link href="/" legacyBehavior>
+        <Link href="/">
             <Comp>Home</Comp>
-        </Link>)
+        </Link>
```

Comment:
We should emit a comment here instead that this needs manual review. The codemod for async params has an example that you can use as a convention.